doc/plugins/inventory/icinga-inventory-plugin.md

```diff
@@ -259,13 +259,13 @@ This inventory plugin needs
 
 **plugin**
 
-This is a token that ensures that the plugin definitions are meant for this inventory plugin.  
+This is a token that ensures that the plugin definitions are meant for this inventory plugin.<br>
 The form is `namespace.collection_name.plugin_name`.
 
 This must be `icinga.icinga.icinga`
 
-Required: `true`  
-Type: `string`  
+Required: `true`<br><br>
+Type: `string`<br><br>
```

Comment:
Please make it just one <br> for each line
```suggestion
Required: `true`<br>
Type: `string`<br>
```

Reply:
Not sure what happened there, probably overeager copypasting